cmd/version.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021 The Sigstore Authors
```

Comment:
I saw this project is missing some header files and other files contain a header with a username.

should we make all the same? how about this
` // Copyright 2021 The Sigstore Authors`?